src/components/Auth/styles.tsx

```diff
@@ -0,0 +1,146 @@
+import { makeStyles, Theme as AugmentedTheme } from '@material-ui/core/styles';
```

Comment:
Is this replacing the scss file? Once its been fully moved over be sure to delete that file